src/__tests__/__fixtures__/explicit-no-ssr/output.js

```diff
@@ -0,0 +1,6 @@
+import { lazyForPaint } from 'react-loosely-lazy';
+const ExplicitNoSsr = lazyForPaint(() => import('./my-component'), {
+  ssr: false,
+  getCacheId: () => require.resolveWeak('./my-component'),
```

Comment:
Isn't `resolveWeak` a webpack feature? This won't be compatible with parcel if we end up supporting that.

Reply:
Yeah, not gonna be compatible right away. We will need to build parcel support into it.